connexion/apps/abstract.py

```diff
@@ -10,7 +10,7 @@
 
 class AbstractApp(metaclass=abc.ABCMeta):
     def __init__(self, import_name, api_cls, port=None, specification_dir='',
-                 host=None, server=None, arguments=None, auth_all_paths=False, debug=None,
+                 host=None, server=None, server_args=dict(), arguments=None, auth_all_paths=False, debug=None,
```

Comment:
This should be `None`
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
I shall submit a new change...